### DISCUSSION TITLE
Monolayer/bilayer pouch cells

### DISCUSSION THREAD

**Discussion by epsilon-smaller-zero** [pybamm-team/PyBaMM#3209] [Q&A]
Hello,

the default geometry of cells are pouch cells. Regarding [ Battery geometry](https://github.com/pybamm-team/PyBaMM/blob/3f7df5fb2cfeddb9ecee6581dac8c70efec06205/pybamm/geometry/battery_geometry.py) it is a monolayer pouch cell. Is there a way to change from monolayer to bilayer?

Secondly, the parameter set "Chen2020" uses a 21700 cell. If I use a pouch cell, how are the values transfered to a pouch cell? I could not find anything.

Thank you in advance,
Carina

**Comment by valentinsulzer** [comment]
The standard SPM/SPMe/DFN models assume we just have a single 1D anode/separator/cathode sandwich. For a jelly roll, the "Electrode height [m]" and "Electrode width [m]" parameters refer to the unwound cell. There is no difference between a section in the middle of the cylinder or at the edge. 

We do also have some work on extending to a [jellyroll](https://iopscience.iop.org/article/10.1149/1945-7111/abd44f/pdf) and [pouch cell](https://iopscience.iop.org/article/10.1149/1945-7111/abbce4/meta)

At the moment there is no "easy" way to change to a bilayer pouch cell and resolve differences between the layers. The way we would typically approach this is pretend it is a monolayer pouch cell that is twice as big.

To convert Chen2020 to a pouch cell, you should start by adjusting the electrode height and width.
